README.md

```diff
@@ -0,0 +1,130 @@
+# <center>DaoJS</center>
+
+## <center>Environment Setup</center>
+
+#### Git:  https://git-scm.com/
+
+#### NodeJS:  https://nodejs.org/en/ 
+
+####  Yarn: https://yarnpkg.com/lang/en/
+
+####  Browser: latest chrome: https://www.google.com/chrome/
```

Comment:
You can remove them because they are not must-have items.

Some other developers may have their own preference of other tools.